src/router.ts

```diff
@@ -345,7 +351,13 @@ export function createRouter({
   ) {
     // a more recent navigation took place
     if (pendingLocation !== toLocation) {
-      return triggerError(new NavigationCancelled(toLocation, from), isPush)
+      return triggerError(
+        createRouterError(ErrorTypes.NAVIGATION_CANCELLED, {
+          from: toLocation,
```

Comment:
isn't it the other way around? `from, to: toLocation`?

Reply:
oops😂